xapian-applications/omega/postprocess.py

```diff
@@ -130,8 +116,10 @@ def generate_combined_log(search_log, clicks_log, final_log):
                         clicklist[index] = '%s:%i' % (did, did_to_count[did])
                     else:
                         clicklist[index] = did + ':0'
-            elif not clicklist:
+            else:
                 for index, did in enumerate(clicklist):
+                    if clicklist[index] == '':
+                        continue
```

Comment:
`hits` can be empty sometimes (e.g. in case of a wrongly worded query text). This check is there basically to avoid entries like `:0` under Clicks field.